winlogbeat/eventlog/wineventlog.go

```diff
@@ -58,6 +58,7 @@ type winEventLogConfig struct {
 	Forwarded     *bool              `config:"forwarded"`
 	SimpleQuery   query              `config:",inline"`
 	NoMoreEvents  NoMoreEventsAction `config:"no_more_events"` // Action to take when no more events are available - wait or stop.
+	IsFile        bool               `config:"is_file"`        // To determine whether to subscribe or read events
```

Comment:
Will this change impact eventlog plugin functionality?

Could you please check if eventlog plugin can be built with this version of library.

I' thought that changes regarding file should go at another iteration of changes in the next PR.

Reply:
This change should not impact event log plugin existing functionality.
To test this version of library with elastic beats we need to update the commit version in event log plugin.
We anyhow would want to test event log plugin with such flag as the method which will be called from event log plugin (Open) would be having differentiation whether to subscribe event or not.

Reply:
I also vote for another one PR with "open" functionality and it this one only should with changes made by Continuum

Reply:
I will revert the change introduced for differentiating call.